tests/unit/s2n_async_pkey_test.c

```diff
@@ -348,7 +346,8 @@ int async_pkey_invalid_input_callback(struct s2n_connection *conn, struct s2n_as
 
 int async_pkey_invalid_complete(struct s2n_connection *conn, struct s2n_blob *signature)
 {
-    FAIL_MSG("Invalid async pkey callback was invoked. The callback should never be invoked if there was an earlier"
+    FAIL_MSG(
```

Comment:
I tried to find a way to make this not change. But I couldn't figure it out. As I worked on this and clang-formatting kept insisting on this formatting I actually came to like it. 🤷

Reply:
I disagree. Did you try adding it to one of the "*Macro" lists? Maybe StatementMacros?

Reply:
Oh interesting! Yeah.

Reply:
That doesn't work.

Reply:
It's not the best solution but adding it to WhitespaceSensitiveMacros does work.

Reply:
AlwaysBreakBeforeMultilineStrings: true was the culprit